dspace/modules/xmlui/src/main/java/org/dspace/app/xmlui/aspect/submission/submit/ProjectStep.java

```diff
@@ -178,7 +179,11 @@ private void renderResults(Division div, Metadatum[] dcValues) throws WingExcept
             }
         } else {
             Division warningDiv = div.addDivision("default-project-warning", "alert alert-warning bold");
-            warningDiv.addPara().addContent(T_default_project_warning.parameterize(defaultProject, defaultFunder));
+
+            // don't show the default project warning if there is no default project configured
+            if(new DSpace().getServiceManager().getServiceByName("defaultAuthorityCreator", DefaultAuthorityCreator.class).hasValidDefaultAuthorityConfiguration()) {
```

Comment:
I think it is easier and less expensive to check if default project or default funder is blank, please refactor.